docs/experimental/ticket-auction.md

```diff
@@ -0,0 +1,402 @@
+                                            (pair (nat %set_reserve_price)
+                                                  (pair (timestamp %set_start_time) (pair (int %set_round_time) (ticket %ticket nat))))))
+                                   (pair (nat %opening_price)
+                                         (pair (nat %reserve_price)
+                                               (pair (timestamp %start_time) (pair (int %round_time) (nat %ticket_id))))))
+                                (nat %burn))
+                            (or (map %mint string bytes) (ticket %receive nat)))
+                        (pair %send (contract %destination (ticket nat)) (nat %ticket_id))) ;
+                  storage
+                    (pair (address %admin)
+                          (pair (big_map %tickets nat (ticket nat))
+                                (pair (nat %current_id) (big_map %token_metadata nat (pair nat (map string bytes)))))) ;
+                  code {...}
+
+New contract KT1EAMUQC1yJ2sRPNPpLHVMGCzroYGe1C1ea originated.
+
+```
+
+Note we only include relevant terminal output.
+
+
+#### Bob
+
+Bob originates his nft wallet and sets himself as the admin
+
+```sh
+$ tezos-client originate contract nft-wallet-bob transferring 0 from bob \
+        running "$(pwd)/michelson/nft_wallet.tz" \
+         --init "Pair $BOB_ADDRESS (Pair {} (Pair 0 {}))" \
+         --burn-cap 1
+
+New contract KT1QQukCBULzFu6samB5FpbLNBXmNzArSpTs originated.
+
+```
+Make a variable for the new contract address:
+
+```shell
+$ ALICE_WALLET_ADDRESS=KT1EAMUQC1yJ2sRPNPpLHVMGCzroYGe1C1ea
+$ BOB_WALLET_ADDRESS=KT1QQukCBULzFu6samB5FpbLNBXmNzArSpTs
+```
+
+Alice and Bob's wallet contracts can be viewed on TzStat's edonet explorer
+at https://edo.tzstats.com/KT1EAMUQC1yJ2sRPNPpLHVMGCzroYGe1C1ea and
+https://edo.tzstats.com/KT1QQukCBULzFu6samB5FpbLNBXmNzArSpTs respectively.
+
+## NFT Auction Contract
+
+A separate NFT auction contract can be used to auction off ones NFTs
+to prospective buyers. We use a [Dutch](https://en.wikipedia.org/wiki/Dutch_auction),
+descending price auction in which the asking price of the NFT is decreased
+until a buyer is found. The NFT auction contract has an entrypoint to
+`configure` the auction, `start` the auction, `drop_price` of the NFT
+after a round has passed, `buy` the NFT by sending the asking price,
+and an entrypoint for the admin to `cancel` the auction
+and send the NFT back to their wallet.  
+
+
+### Auction
+
+Alice originates nft-auction contract and initializes several storage parameters.
+
+```sh
+"Pair
+  (Pair
+    $ALICE_ADDRESS      -- Alice set as admin
+    (Pair
+      0                 -- current_price set to 0 as default
+      (Pair
+        0               -- reserve_price set to 0
+        (Pair
+          False         -- in_progress set to False as no auction in progress
+          (Pair
+            0           -- start_time set to 0 as default
+            0           -- round_time set to 0 as default
```

Comment:
You don't mention the units (seconds) until later: maybe move that description up to the first usage or add units here?

Reply:
Good catch. I will add units here